.github/workflows/notify-teams.yml

```diff
@@ -0,0 +1,41 @@
+permissions:
+  contents: read
+name: Notify Teams on UI PR Creation
```

Comment:
A suggestion: was wondering id it could be useful for other teams? Would be great if we could map a `path` to a `TEAMS_WEBHOOK_URL` like
```
'web_ui/**' - UI_TEAMS_WEBHOOK_URL
'interactive_ai/**' - IAI_TEAMS_WEBHOOK_URL
'platform/**' - PLATFORM_TEAMS_WEBHOOK_URL
```